content/documentation/quickstart.md

```diff
@@ -53,26 +53,34 @@ This will run the NATS Server on port 4222 in debug mode. Debug mode isn't *nece
 
 ## Send messages to NATS using a client {#send}
 
-You can find instructions for your language of choice in the box below:
+All NATS messages can include the following:
+
+* A target subject
+* A reply subject (optional)
+* A message payload (as an array of bytes)
+
+The language-specific examples below show you how to send a message with a target subject `updates` and a message payload of `All is well`:
 
 {{< partial "doc/publish_bytes.html" >}}
 
 > For more comprehensive instructions, see the [Sending Messages](/documentation/writing_applications/publishing) guide.
 
 ## Receive messages from NATS using a client {#receive}
 
-Messages from NATS can be received either [synchronously](#synchronous) or [asynchronously](#asynchronous).
+NATS clients receive messages from NATS by establishing *subscriptions*. With subscriptions, the client notifies the NATS server about which subject the client is interested in and then listens for messages with that subject. Messages from NATS can be received either [synchronously](#sync) or [asynchronously](#async).
 
-> For more comprehensive instructions, see the [Receiving Messages](/documentation/writing_applications/subscribing/) guide.
+The language-specific examples below show you how to receive messages with the subject `updates` both [synchronously](#sync) and [asynchronously](#async).
 
-### Receive messages synchronously
+> For more comprehensive instructions, see the [Receiving Messages](/documentation/writing_applications/subscribing/) guide.
 
-You can find instructions for your language of choice in the box below:
+### Receive messages synchronously {#sync}
 
 {{< partial "doc/subscribe_sync.html" >}}
 
-### Receive messages ascnchronously
+### Receive messages ascnchronously {#async}
+
+{{< partial "doc/subscribe_async.html" >}}
 
-You can find instructions for your language of choice in the box below:
+## Next steps
 
-{{< partial "doc/subscribe_async.html" >}}
\ No newline at end of file
+Congrats! You've now successfully used NATS as a message broker using a client in your preferred language. But there's much more to explore in NATS.
```

Comment:
@ColinSullivan1 what might be a good next step? "Try blah blah blah..."

Reply:
What would you prefer to point users to?